composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "phpunit/phpunit": "^7.0",
```

Comment:
How about defining `^7.0 || ^8.0` for two versions?

It can support the different PHP versions.